plugins/modules/foreman_compute_resource.py

```diff
@@ -71,78 +72,67 @@
 '''
 
 EXAMPLES = '''
-- name: vmware compute resource
+- name: Create livirt compute resource
```

Comment:
I think I'd leave the vmware and ovirt create examples, just to show how that works with different CRs